styles/globals.css

```diff
@@ -2,6 +2,22 @@
 @tailwind components;
 @tailwind utilities;
 
+::-webkit-scrollbar {
+  width: .5rem;
+}
+
+::-webkit-scrollbar-track {
+  background: #00000033;
```

Comment:
Tailwind already have some color variables, please use those. Also, you can define css variables